Samples/DirectMLSuperResolution/DirectMLSuperResolution.vcxproj

```diff
@@ -40,58 +40,58 @@
     <ProjectGuid>{70CDBD87-F286-4EE0-87F1-5A1D09396CDA}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>DirectMLSuperResolution</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v143</PlatformToolset>
```

Comment:
>v143 [](http://example.com/codeflow?start=21&length=4)

I think this corresponds to VS2022. Is this intentional? Our samples should build fine on VS2017 and above, I don't think there's any particular reason to require VS2022.

Reply:
Yeah, we generally leave samples no higher than needed so users with older VS's can easily consume the samples. Anyone with a newer VS can trivially upgrade it if they want, but since the Nuget contains the DirectML.h file anyway, there's no need for a newer SDK.

Reply:
Yeah, this was intentional. I was thinking other way around, like we should always target latest Windows SDK. But this reasoning makes sense.